allow `filePath` input

lunelson opened the pull request:
This PR supports the use-case where a site is being built to a subdirectory: while HTML file paths should be evaluated from the root of the build, the actual `sitemap.xml` file should be placed in the subdirectory, since in such cases the root directory of the deploy site will be inaccessible and probably already has a sitemap. 

This enables a `filePath` input parameter, passed to the sitemap library's `fileName` parameter. it defaults to `sitemap.xml` if not passed, but allows intervention for the case described above, if needed

erezrokah reviewed (changes requested):
Hi @lunelson and sorry for the very late reply on this.
Can you:
1. Document this change in the README?
2. Create this PR from a dedicate feature branch instead of `master`? This makes it easier for maintainers to push changes.

lunelson commented:
Hi @erezrokah thanks for your reply and sorry for my own late reply. I've moved my changes to a branch and added some notes to the README, but I can't seem to change the origin of this PR so I'm closing and re-creating: https://github.com/netlify-labs/netlify-plugin-sitemap/pull/50
